src/data/topics.js

```diff
@@ -0,0 +1,25 @@
+export const topics = [
+  { value: 'Градска среда - Чист и зелен град', label: 'Чист и зелен град' },
+  { value: 'Градска среда - Иновации', label: 'Иновации' },
+  { value: 'Градска среда - Инфраструктура', label: 'Инфраструктура' },
+  { value: 'Градска среда - Жилищна политика', label: 'Жилищна политика' },
+  { value: 'Социална политика - Образование', label: 'Образование' },
+  { value: 'Социална политика - Здраве и спорт', label: 'Здраве и спорт' },
+  {
+    value: 'Транспортна инфраструктура - Градски транспорт',
+    label: 'Градски транспорт',
+  },
+  {
+    value: 'Транспортна инфраструктура - Паркинги и гаражи',
+    label: 'Паркинги и гаражи',
+  },
+  { value: 'Транспортна инфраструктура - Ремонти', label: 'Ремонти' },
+  { value: 'Администрация - Бюджет и финанси', label: 'Бюджет и финанси' },
+  { value: 'Бизнес и туризъм', label: 'Бизнес и туризъм' },
+  {
+    value: 'Култура и историческо наследство',
+    label: 'Култура и историческо наследство',
+  },
+  { value: 'Събития - Акции', label: 'Акции' },
+  { value: 'Събития - Срещи', label: 'Срещи' },
+];
```

Comment:
```suggestion
export const topics = [
  { value: 'Градска среда - Чист и зелен град', label: 'Чист и зелен град' },
  { value: 'Градска среда - Иновации', label: 'Иновации' },
  { value: 'Градска среда - Инфраструктура', label: 'Инфраструктура' },
  { value: 'Градска среда - Жилищна политика', label: 'Жилищна политика' },
  { value: 'Социална политика - Образование', label: 'Образование' },
  { value: 'Социална политика - Здраве и спорт', label: 'Здраве и спорт' },
  { value: 'Транспортна инфраструктура - Градски транспорт', label: 'Градски транспорт' },
  { value: 'Транспортна инфраструктура - Паркинги и гаражи', label: 'Паркинги и гаражи' },
  { value: 'Транспортна инфраструктура - Ремонти', label: 'Ремонти' },
  { value: 'Администрация - Бюджет и финанси', label: 'Бюджет и финанси' },
  { value: 'Бизнес и туризъм', label: 'Бизнес и туризъм' },
  { value: 'Култура и историческо наследство', label: 'Култура и историческо наследство' },
  { value: 'Събития - Акции', label: 'Акции' },
  { value: 'Събития - Срещи', label: 'Срещи' }
];
```
Removing some unnecessary commas.

Reply:
this is prettier formatting setting already in place/accepted